src/web/crate_details.rs

```diff
@@ -185,6 +185,18 @@ impl CrateDetails {
             default_target: rows.get(0).get(25),
         };
 
+        let doc_targets = {
+            let data: Json = rows.get(0).get(22);
```

Comment:
This will now panic if `doc_targets` is null, can you add a migration making it `NOT NULL`? I checked manually and there's currently no releases with a null entry.